projects/media-viewer/src/lib/utils/mouse-selection-utils.ts

```diff
@@ -0,0 +1,41 @@
+import { Injectable } from '@angular/core';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class MouseSelectionUtils {
+
+  // Return the absolute positioning of the top, left x & y coordinates relative to the localElement
+  static getTargetElementAbsoluteMouseCoordinates(event: MouseEvent): { x: number, y: number } {
```

Comment:
Can you extract that to an interface below: 

```
export interface Coordinates {
  x: number,
  y: number
}```

Then you can just use `: Coordinates`

Reply:
yep, sounds sensible. Can't see any other type definitions in our project currently. Any recommendations on a sensible home that is shared?

Reply:
The same file is fine for now

Reply:
done